src/software/ai/hl/stp/evaluation/robot.h

```diff
@@ -46,5 +45,9 @@ namespace Evaluation
      * @param timestamp The time at which we want to know if the robot had the ball
      * @return True if the ball is close to the front dribbler and false otherwise
      */
-    bool robotHasPossession(Ball ball, Robot robot, Timestamp timestamp = Timestamp());
+    bool robotHasPossession(const Ball& ball, const Robot& robot,
+                            Timestamp timestamp = Timestamp());
+
+    bool robotBeingPassedTo(const World& world, const Robot& robot,
+                            Timestamp timestamp = Timestamp());
```

Comment:
We should make sure this function is tested

Reply:
Going to create an issue for this

Reply:
Oops left comment on wrong thread, added a few tests for this function